rootfs/Dockerfile

```diff
@@ -3,13 +3,14 @@ FROM gcr.io/google_containers/ubuntu-slim:0.4
 ENV TERM=xterm
 
 # disable source repos (speeds up apt-get update)
-RUN sed -i -e 's/^deb-src/#deb-src/' /etc/apt/sources.list && \
-apt-get update && \
-apt-get install -y \
-	bash \
-	curl \
-	net-tools \
-	procps && \
-	apt-get clean && \
-	rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/man /usr/share/doc && \
-bash -c "mkdir -p /usr/share/man/man{1..8}"
+RUN sed -i -e 's/^deb-src/#deb-src/' /etc/apt/sources.list \
+	&& apt-get update \
+	&& apt-get upgrade -y \
```

Comment:
`dist-upgrade` does apparently pull in some other (optional?) packages, so the container image is somewhat bigger:
```bash
$ docker images
REPOSITORY               TAG                    IMAGE ID            CREATED              SIZE
deis/base                apt-get-dist-upgrade   c5676a90a049        About a minute ago   103.3 MB
deis/base                apt-get-upgrade        51c6d6424ab9        19 minutes ago       83.74 MB
deis/base                0.3.1                  b1fe89b6a4fd        20 minutes ago       78.27 MB
```
That may be fine. I was just curious to see.